static/js/blueslip.ts

```diff
@@ -194,46 +202,52 @@ export function exception_msg(ex) {
 }
 
 $(window).on("error", (event) => {
-    const ex = event.originalEvent.error;
-    if (!ex || ex instanceof BlueslipError) {
```

Comment:
We still need this check for `BlueslipError` to avoid duplicate reports from `blueslip.error()` in development.

(Or we could modify `blueslip.error` so it doesn’t `report_error` in development itself. The `BlueslipError` class predates this `error` event handler and probably isn’t needed anymore.)

Reply:
I went with a simple fix and added back the `ex instanceof BlueslipError` check.